todoList.js

```diff
@@ -242,10 +243,10 @@ function changePriority(e, index) {
     let value = e.target.value;
 
     switch(value) {
-        case "":
+        case "no-value":
             // Clear all classes
             priorityElement.classList.remove("priority-one", "priority-two", "priority-three");
-            item.priority = "";
+            item.priority = "no-value";
```

Comment:
To simplify the switch statement, we should be assigning the `value` variable, than manually assign the value per case.